### DISCUSSION TITLE
FireflyIII breaks when installing - Sender address rejected: not logged in

### DISCUSSION THREAD

**Discussion by ITCMD** [firefly-iii/firefly-iii#7693] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Fresh install of Firefly on Yunohost 11.1.21.4.

When launching the app it wants me to register. When I attempt to register an account it responds with:

```
FireflyIII
Whoops! An error occurred.
Unfortunately, this error was not recoverable :(. Firefly III broke. The error is:

Expected response code "250/251/252" but got code "553", with message "553 5.7.1 <myusername@domainhidden>: Sender address rejected: not logged in".

This error occured in file /var/www/firefly-iii/app/Jobs/MailError.php on line 103 with code 0.

More information
Please collect more information in the storage/logs directory where you will find log files. If you're running Docker, use docker logs -f [container]. You can read more about collecting error information in the FAQ.

Get help on GitHub
You're more than welcome to open a new issue on GitHub.

Use the search!
Include the information from this debug page.
Tell us more than "it says Whoops!"
Include error logs (see above).
Tell us what you were doing.
```

### Debug information

I cannot access the debug page without logging in, and I cannot login

### Expected behaviour

Creates an account

### Steps to reproduce

1. Install Firefly
2. Attempt to set up a user

### Additional info

ff3-fcgi logs: https://paste.debian.net/1284283/
ff3-cli logs: https://paste.debian.net/1284284/
ff3-audit logs (personal data removed):
```
[2023-06-27T10:43:14.923311-04:00] local.INFO: AUDIT: Show login form (1.1). (my.ip.address -> GET:https://yunohost.mydomain.com/firefly/login) [] []
[2023-06-27T10:43:19.935712-04:00] local.INFO: AUDIT: Show login form (1.1). (my.ip.address -> GET:https://yunohost.mydomain.com/firefly/login) [] []
[2023-06-27T10:50:50.763454-04:00] local.INFO: AUDIT: Show login form (1.1). (my.ip.address -> GET:https://yunohost.mydomain.com/firefly/login) [] []
[2023-06-27T10:51:01.283860-04:00] local.INFO: AUDIT: User is trying to login using "sig_firefly@mydomain.com" (my.ip.address -> POST:https://yunohost.mydomain.com/firefly/login) [] []
[2023-06-27T10:51:01.356303-04:00] local.INFO: AUDIT: User "sig_firefly@mydomain.com" has been logged in. (my.ip.address (sig_firefly@mydomain.com) -> POST:https://yunohost.mydomain.com/firefly/login) [] []
[2023-06-27T10:53:55.496804-04:00] local.INFO: AUDIT: Show login form (1.1). (my.ip.address -> GET:https://yunohost.mydomain.com/firefly/login) [] [] 
```

**Comment by JC5** [comment]
That looks like an error in your mail settings. What are they?

**Reply by ITCMD** [reply]
I dont have any mail settings, I just installed firefly and have not been able to register to begin setup.

If you mean Yunohost, I have no special settings enabled.

**Reply by JC5** [reply]
Then I don't know, I don't know anything about Yunohost.

**Reply by ITCMD** [reply]
Does it require an active existing email server to be active before installation? If so, where should I have the config for that?

All Yunohost does is run scripts that take care of the normal setup.

**Reply by ITCMD** [reply]
could it be because my domain is a non-average email address? It's @myfirstname-mylastname.com

**Reply by ITCMD** [reply]
Also "occurred" is missing an r in the error message FYI :)

**Reply by JC5** [reply]
> Does it require an active existing email server to be active before installation? If so, where should I have the config for that?

No it doesn't. That would be a little weird, imo. I don't know where Yunohost keeps its config though.

> could it be because my domain is a non-average email address? It's @myfirstname-mylastname.com

Also no, any email address is accepted.

> Also "occurred" is missing an r in the error message FYI :)

Nicely spotted, i'll pick it up.
